endpoint.go

```diff
@@ -617,11 +475,8 @@ func (ep *endpoint) deleteEndpoint() error {
 }
 
 func (ep *endpoint) addHostEntries(recs []etchosts.Record) {
```

Comment:
Shouldn't these apis be moved to Sandbox since these should be done only once during sandbox creation? This applies to all the container config state which was earlier managed in endpoint